test/command/suite/token_filters/stem/keep_original/add.test

```diff
@@ -0,0 +1,14 @@
+#@on-error omit
+plugin_register token_filters/stem
+#@on-error default
+
+table_create Memos TABLE_NO_KEY
+column_create Memos content COLUMN_SCALAR ShortText
+
+table_create Terms TABLE_PAT_KEY ShortText \
+  --default_tokenizer TokenBigram \
```

Comment:
I wondered why the stemming works when the tokenizer is Bigram, but I noticed that `Tokenbigram` uses white-space-separate for ascii when a normalizer is used. 
 (https://groonga.org/docs/reference/tokenizers/token_bigram.html)
This is the reason why it works, right?

Reply:
Right.